lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -168,7 +168,7 @@ public void Run()
             }
             catch (Exception e)
             {
-                runtimeException = Optional<Exception>.Of(new SystemException("Caught Exception in clock, failing the Evaluator.", e));
+                runtimeException = Optional<Exception>.Of(new WakeRuntimeException("Caught Exception in clock, failing the Evaluator.", e));
```

Comment:
The code location is in Wake but this exception is not actually Wake-specific; it means that something has gone wrong somewhere in REEF and a more specific catch hasn't caught this.